include/syscall.h

```diff
@@ -122,16 +122,6 @@ typedef u32_t (*_k_syscall_handler_t)(u32_t arg1, u32_t arg2, u32_t arg3,
  */
 static inline bool z_arch_is_user_context(void);
```

Comment:
Could you move this declaration below, as well? So it is close to the _is_user_context definition

Reply:
Done